src/components/ItemEditorPage/CenterPanel/CenterPanel.tsx

```diff
@@ -108,7 +118,7 @@ export default class CenterPanel extends React.PureComponent<Props, State> {
           wheelZoom={1.5}
           wheelStart={100}
           onUpdate={() => this.setState({ isLoading: true })}
-          onLoad={() => this.setState({ isLoading: false })}
+          onLoad={() => this.handleWearablePreviewLoad()}
```

Comment:
small suggestion to simplify and avoid creating a new fn in each render.

```suggestion
          onLoad={this.handleWearablePreviewLoad}
```